app/app/User.php

```diff
@@ -9,89 +10,91 @@
+        'email_verification_token','email_verification_time'
     ];
-	public $timestamps = false;
-	
-	protected $table = 'user';
-	/**
-	 * Indicates if the IDs are auto-incrementing.
-	 *
-	 * @var bool
-	 */
-	public $incrementing = false;
+    public $timestamps = false;
+    
+    protected $table = 'user';
+    /**
+     * Indicates if the IDs are auto-incrementing.
+     *
+     * @var bool
+     */
+    public $incrementing = false;
+
+    public function __construct()
+    {
+        $this->attributes = array('id' => Uuid::generate(4)->string);
+    }
+    
+    public function roles()
+    {
+        return $this->belongsToMany(Role::class, 'user_role');
+    }
+    
+    // Checks if this user has the specified role
+    public function hasRole(int $role_id)
+    {
+        $matches = DB::table('user_role')
+            ->where('user_id', '=', $this->id)
+            ->where('role_id', '=', $role_id)
+            ->first(['id']);
+        return !!$matches;
+    }
+    
+    /**
+    isQuestionRequired checks if the specified id is in the array of questions.
+    This is used in profile.blade.php and ProfileController.
+
+    An alternative was to have a method that queries the database for each
+    individual question but this seemed much less efficient than getting a
+    complete list of questions and using PHP code to look for ids in that array.
 
-	public function __construct() {
-		$this->attributes = array('id' => Uuid::generate(4)->string);
-	}
-	
-	public function roles()
-	{
-	   return $this->belongsToMany(Role::class, 'user_role');
-	}
-	
-	// Checks if this user has the specified role
-	public function hasRole(int $role_id)
-	{
-		$matches = DB::table('user_role')
-			->where('user_id', '=', $this->id)
-			->where('role_id', '=', $role_id)
-			->first(['id']);
-		return !!$matches;
-	}
-	
-	/**
-	isQuestionRequired checks if the specified id is in the array of questions.
-	This is used in profile.blade.php and ProfileController.
-	
-	An alternative was to have a method that queries the database for each
-	individual question but this seemed much less efficient than getting a
-	complete list of questions and using PHP code to look for ids in that array.
-	
-	@param required_questions should be an array of Question instances.
-	@param id should be a question's id
-	*/
-	public static function isQuestionRequired($required_questions, $id)
-	{
-		foreach ($required_questions as $question)
-		{
-			if ( $question->id === $id )
-				return true;
-		}
-		return false;
-	}
-	
-	public function home_country()
-	{
+    @param required_questions should be an array of Question instances.
+    @param id should be a question's id
+    */
+    public static function isQuestionRequired($required_questions, $id)
+    {
+        foreach ($required_questions as $question) {
+            if ($question->id === $id) {
+                return true;
+            }
+        }
+        return false;
+    }
+    
+    public function homeCountry()
```

Comment:
I like the change that your PSR-2 tool made here with the case but the tool didn't fix all references to the old name which lead to a bug you can see from the following screenshot:

![image](https://user-images.githubusercontent.com/11838191/42116102-d3fd8796-7bc3-11e8-8954-d7cef14695db.png)


Run the tests in tests/*.php to see.  There are instructions on how to run them in app/readme.md (https://github.com/hhaccessibility/hhaccessibility.github.io/blob/master/app/readme.md).

Reply:
@joshi1983 I need to look this!